### PR TITLE
Distance users

### PR DESCRIPTION
**Pull request by touhamiabi**
Now our events and partners index have geocoding and sorted in ascending order

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -1,11 +1,26 @@
...
+  addresses = JSON.parse(File.read(path_to_file))
+  return quantity ? addresses.sample(quantity) : addresses unless format == 'string'
+
+  addresses.map do |address|
+    "#{address['number']} #{address['street']}, #{address['city']}, #{address['place']}, #{address['country']}"
+  end
+end
+
+ADDRESS = get_mtl_addresses_from_json(50)
+pp ADDRESS.first
```
Comment (ShadowSpin1) [inline db/seeds.rb]: do you need this line or it was just for testing?

File: db/schema.rb [hunk 2]
```diff
@@ -133,6 +133,8 @@
     t.string "description"
     t.string "gender"
     t.string "training_style"
+    t.float "latitude"
```
Comment (ShadowSpin1) [inline db/schema.rb:136]: are we meant to have these two lines this time? last time, we had to remove these two lines manually.

Comment (laura-gonzalez-z) [reply db/schema.rb:136]: Yes! This time it's normal to have them since there's a migration creating them 👍

File: db/seeds.rb (hunk 1 above)
Comment (laura-gonzalez-z) [reply db/seeds.rb]: Maybe it was added when debugging the file with Ryan. Let's wait for @touhamiabi to confirm or remove it.

Comment (touhamiabi) [reply db/seeds.rb]: thanks! yes I need to remove them

**Review by laura-gonzalez-z — APPROVED** [review]

**Pull request merged**